### PR TITLE
Upgrade branch 2.7.x using TemplateControl

### PR DESCRIPTION
**Pull request by octonato**
```
Updated with template-control on 2019-01-10T23:13:51.679Z
  /LICENSE:
    wrote /LICENSE
  /NOTICE:
    wrote /NOTICE
  /.travis.yml:
    wrote /.travis.yml
  **build.sbt:
    scalaVersion := "2.12.8"
  **/build.properties:
    sbt.version=1.2.8
  **/plugins.sbt:
    addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.0-RC9")

```

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -5,7 +5,7 @@ version := "1.0"
 lazy val `play-webgoat` = (project in file(".")).enablePlugins(PlayScala)
 
 crossScalaVersions := Seq("2.12.7", "2.11.12")
-scalaVersion := crossScalaVersions.value.head
+scalaVersion := "2.12.8"
```
Comment (SethTisue) [inline build.sbt:8]: @renatocaval the inconsistency between scalaVersion and crossScalaVersions here is odd and presumably unintended

Comment (octonato) [reply build.sbt:8]: sure, I need to review it. 

We update those projects using TemplateControl (http://github.com/lightbend/templatecontrol) and this particular project is not really aligned and it's causing this kind of problem. 

TemplateControl sees a line with `scalaVersion :=` and decides to replace it with what it has configured. 

I will create an issue on TemplateControl so we don't forget to find a solution for this.

Comment (wsargent) [reply build.sbt:8]: @renatocaval 	you can use `tc-skip` on that line to ignore it 

https://github.com/lightbend/templatecontrol/blob/master/src/main/scala/templatecontrol/Task.scala#L99

Comment (octonato) [reply build.sbt:8]: Absolutely, especially because tc-skip was my idea. How could I have forgotten it? ;-)

**Pull request merged**